OpenRA.Game/System.Diagnostics.CodeAnalysis/MemberNotNullAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Workaround for mono where this type is internal.
```

Comment:
Required otherwise mono gives `error CS0122: 'MemberNotNullAttribute' is inaccessible due to its protection level`.

Also note the licence.